### ISSUE TITLE
[update] skin.titan

### ISSUE THREAD

**Issue by marcelveldt** [xbmc/repo-skins#54]
### Add-on details:
- Add-on name: Titan 
- Add-on ID: skin.titan
- Version number: 3.7.49
- Kodi/repository version: KRYPTON
- URL: https://github.com/marcelveldt/skin.titan
- Revision/tag: latest
- Branch: krypton

### Description of changes:
- Compatability with new skinhelper addons.
- code cleanup/refactor
- smaller footprint because all static content is moved to resource addons.

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-skins/blob/master/CONTRIBUTING.md) document

**Comment by marcelveldt** [comment]
Hi @ronie do you want me to submit a PR for the skin update or is it fine this way ?

**Comment by ronie** [comment]
updated, thx!
https://github.com/xbmc/repo-skins/commit/aad2b271b868f6499aeb4427d3b157271ab4e8a6

**Comment by marcelveldt** [comment]
@ronie, Thanks for adding the whole bunch of PR's!

I'm very sorry but I just discovered a very stupid error, possible caused by some vague merge error in Git after the rename of those resource addons...

In the addon.xml of the krypton version of the skin, the xbmc gui version was somehow reverted to 5.10 instead of 5.12. I completely missed that and I guess you too while processing the PR.

I just fixed the issue on Git, is it still possible to fix this ?

Again, very sorry, hopefully it's an easy fix.
